API/c/Src/analog_in.c

```diff
@@ -0,0 +1,65 @@
+#include "analog_in.h"
+
+ADC_HandleTypeDef ADC_InitStruct;
+
+const int REFRESH_PERIOD = 10;
+const int ADC_MAX = 4096;
+const float ADC_RATIO = 3.3 / 4096;
```

Comment:
Is ADC_MAX 1<<12? And is the max voltage read by the ADC 3.3?

Reply:
Yup do I need to document that? I honestly dont get why people write constants as shifts to show 2 ^ N exponents instead of just the number